combat-trainer.lic

```diff
@@ -2207,8 +2242,9 @@ class TrainerProcess
     return unless Time.now - UserVars.almanac_last_use >= 600
     return if left_hand && !game_state.currently_whirlwinding
     unless @almanac_skills.empty?
-      training_skills = @almanac_skills.select { |skill| DRSkill.getxp(skill) < 18 }
-      training_skill = game_state.sort_by_rate_then_rank(training_skills).first
+      training_skill = @almanac_skills
+                       .select { |skill| DRSkill.getxp(skill) < 18 }
+                       .sort_by { |skill| DRSkill.getxp(skill) }.first
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept